Release note for run details and logs improvements (#3428)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-versions/release-notes/08-May-2023/run-details-and-logs-improvements.md

---
title: "Run details and logs usability and design improvements"
sidebar_label: "Update: Improvements to run details and logs"
sidebar_position: 3
tags: [May-26-2023, Scheduler]
---

> New usability and design improvements to the run details and logs in dbt Cloud are now available. The ability to triage errors in logs is a big benefit of using dbt Cloud's job and scheduler functionality. These updates help make the process of finding root causes much easier.

Reviewer: ```suggestion
New usability and design improvements to the run details page and logs in dbt Cloud are now available. The ability to triage errors in logs is a big benefit of using dbt Cloud's job scheduler functionality. These updates help make the process of finding root cause much easier.
```

Author: got it! suggest adding the definite article:

```suggestion
New usability and design improvements to the run details and logs in dbt Cloud are now available. The ability to triage errors in logs is a big benefit of using dbt Cloud's job and scheduler functionality. These updates help make the process of finding the root cause much easier.
```

Highlights include:

- Surfacing a warn state on a run step
- Search in logs
- Easier discoverability of errors and warnings in logs

> - Lazy loading of logs, making the whole page load faster and feel more performant

Reviewer: ```suggestion
- Lazy loading of logs, making the whole run details page load faster and feel more performant
```
